src/redux.js

```diff
@@ -1191,8 +1198,8 @@ export function getPanelButtons(state) {
     next = isPanelAvailable(state, "dataDisplayLabel")
       ? { panel: "dataDisplayLabel", text: "Continue" }
       : isPanelAvailable(state, "dataDisplayFeatures")
-      ? { panel: "dataDisplayFeatures", text: "Continue" }
-      : null;
+        ? { panel: "dataDisplayFeatures", text: "Continue" }
+        : null;
```

Comment:
Why did this indenting change?

Reply:
I formatted the file with Prettier. I changed it back.